content/hi/docs/reference/scheduling/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: निर्धारण
```

Comment:
```suggestion
title: निर्धारण
```
I'd prefer seeing transliteration instead of translation here. Scheduling is a technical term in Kubernetes context which may lose its meaning or become difficult to understand when translated.

Transliteration is changing just script while keeping the pronunciation same which would be `शेडुलिंग ` here.